util.go

```diff
@@ -144,3 +144,9 @@ func hasPriorityErrors(errorMessages map[string]LogMessage, level log.Level) boo
 	}
 	return false
 }
+
+// getLocale tries to get current locale
+func getLocale() string {
+	locale := os.Getenv("LANG")
```

Comment:
This should be more complicated. Not sure about using https://github.com/Xuanwo/go-locale

Reply:
Yes, there is a list of environment variables that gettext checks: https://www.gnu.org/software/gettext/manual/html_node/The-LANGUAGE-variable.html

Also, please note that `LANGUAGE` can be a colon-separated list.

Reply:
I think this is enough for the current use-case. We'll need to write more once we support localization of rhc itself; until then I'm fine with this solution.

The scope of this PR is to send the locale over the D-Bus, which works. Proper POSIX-compliant locale detection is out of scope of the PR in my opinion.

Reply:
It seems that `go-locale` supports all env. vars. described in https://www.gnu.org/software/gettext/manual/html_node/The-LANGUAGE-variable.html. The `go-locale` also supports every possible OS from `AIX` to `z/OS`. Maybe, it is overkill. Not sure.

We are going to support localization of `rhc` in better way. Some other go module used for localization could possibly provide similar/same functionality as `go-locale`.

I think that it would be good to defer decision how to detect locale for this case after proper localization is implemented for `rhc`.

I will add `FIXME` comment to the code, and open issue or card. Sounds good?